packages/eslint-plugin-baseui/__tests__/deprecated-theme-api-test.js

```diff
@@ -0,0 +1,256 @@
+/*
+Copyright (c) 2018-2020 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+const RuleTester = require('eslint').RuleTester;
+const rule = require('../src/deprecated-theme-api.js');
+const MESSAGES = require('../src/messages.js');
+
+RuleTester.setDefaultConfig({
+  parser: require.resolve('babel-eslint'),
+  parserOptions: {
+    ecmaVersion: 6,
+    sourceType: 'module',
+  },
+});
+
+const tests = {
+  valid: [
+    {
```

Comment:
does the code below pass or fail?

```
const colors = {background: 'xyz'};
function MyComponent() {
  return <p style={{backgroundColor: colors.background}}>hello</p>
}

Reply:
Currently `colors.background` will be flagged.

Reply:
Fixed in refactor